app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,31 @@
+class RecipesController < ApplicationController
+  def index
+    @recipes = Recipe.all
+  end
+
+  def show
+    @recipe = Recipe.find(params[:id])
+  end
+  
+  def new
+    @recipe = Recipe.new
+  end
+
+  def create
+    @recipe = Recipe.new(recipe_params)
+    if @recipe.save
+      flash[:success] = "Recipe successfully created"
+      redirect_to recipe_path(@recipe)
+    else
+      flash[:error] = "Something went wrong"
+      render 'new'
+    end
+  end
+  
+  
+  private
+  def recipe_params
```

Comment:
Add a line break after "private". Rubocop should warn about this... is rubocop set up? You can test to run rubocop from the command line, just type  `rubocop`.

Reply:
it is now :)